### PR TITLE
Avoid ambiguous use of 'Panache' in 'Logging with Panache'

### PR DESCRIPTION
**Pull request by holly-cummins**
I'm a bit unsure about the meaning of the heading in https://quarkus.io/guides/logging#logging-with-panache. 

image

Is this a logging technique that only works if a Panache dependency is present? There's no other mention of Panache in the section. Or are we just using 'Panache' as a generic term?

I can see that the change was [introduced on September 1st, 2021](https://github.com/quarkusio/quarkus/commit/a5424aea5f96e6aa5d27eb51750f4efc11597861). I can find references to Panache in 2019, so it's not like the word 'Panache' was chosen before we had a feature called 'Panache'. 

However, I can see examples of log injection being used in projects which don't depend on Panache, for example, in the infinispan client - [usage](https://github.com/quarkusio/quarkus/blob/26cb1bd3f327606f58fe897e48d2e2c87b0198c2/extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/devconsole/InfinispanJsonRPCService.java#L4) and [pom](https://github.com/quarkusio/quarkus/blob/26cb1bd3f327606f58fe897e48d2e2c87b0198c2/extensions/infinispan-client/runtime/pom.xml). 

The `logging-panache` project does depend on panache, and at least some of the tests do only seem to be testing the log injection. 

So I really can't tell:
- If we were using 'Panache' to mean something other than our data access extensions
- If Panache is an implementation detail that users can ignore, even at the dependency level (in which case maybe we don't mention it?)
- If historically we depended on Panache to make this work but now it's just in core (in which case we should remove references)
- This only works in projects with Panache, in which case we should explicitly warn people they also need Panache

I'm assuming it's one of the first three, and I've removed the Panache references. I've gone for a literal substitution of a similar word (ie, assuming the first), but we can reword if it's the second or third.

### CODE REVIEW

**Comment by holly-cummins** [comment]
Or did we use 'with Panache' as a generic 'brand' for our boilerplate-reduction developer joy, but now it's mostly associated with Hibernate ORM?

**Comment by Ladicek** [comment]
> Or did we originally use 'with Panache' as a generic 'brand' for our boilerplate-reduction developer joy

That is exactly how the "with Panache" moniker was supposed to be used, yes.

> but now in external usage it's mostly associated with Hibernate ORM?

That is exactly how the "Panache" moniker is used outside of the inner circle, yes. (Well, not exactly, but almost. We have Hibernate ORM with Panache, Hibernate Reactive with Panache, MongoDB with Panache, and REST Data with Panache. All related to data persistence.)

Also, my personal on the word "Panache" ... cannot be published here, sorry. I would be very happy to replace "Logging with Panache" with "Simplified logging", frankly.

CC @FroMage

**Comment by holly-cummins** [comment]
> > Or did we originally use 'with Panache' as a generic 'brand' for our boilerplate-reduction developer joy
> 
> That is exactly how the "with Panache" moniker was supposed to be used, yes.

Got it. It's a very nice idea, but I think that ship has sailed. It's a shame, since having a shorthand way to refer to boilerplate-reduction-and-developer-joy would be nice. 
 
> > but now in external usage it's mostly associated with Hibernate ORM?
> 
> That is exactly how the "Panache" moniker is used outside of the inner circle, yes.

A quick google confirms that Panache is treated as a [data access library in the general usage ](https://www.google.com/search?client=firefox-b-e&q=quarkus+panache). 

> I would be very happy to replace "Logging with Panache" with "Simplified logging", frankly.

I think that's clean and nice. I did the 'smallest change' PR, changing it to 'with style', but that might be the worst of both worlds. It doesn't communicate what's different, and it doesn't even convey what the benefit is. I'll redo to 'simplified'. 

(I had a quick thesaurus check to see if there was an alternative to 'simplify' that was distinctive enough to act as a 'brand word', but well-known enough to convey meaning. TL;DR, not one that thesaurus.com knew about. :) 

image

"Boiled Down Logging" or "Chastened Logging," anyone? Or maybe "Disinvolved Logging"? 

(As a native speaker, I can confirm that I'm completely puzzled by the inclusion of 'chastened', and I don't think I have ever heard anyone use 'disinvolved' in a sentence.)

The results for 'easy' are a bit more promising:

image

We *could* say "Effortless logging". I slightly prefer that to "Simplified logging," but don't have strong opinions either way. WDYT? It's conveys the value a bit more than 'simplified' (good), but is a bit more marketing-y and less factual (bad, especially in a reference document). 

Actually, I think I've just talked myself out of 'effortless', at least in this context. Update pushed.

**Comment by Ladicek** [comment]
As a non-native speaker, I don't have a good alternative to "simplified" :-/ As a huge fan of Rich Hickey's _Simple Made Easy_, I'm well aware of the distinction between "easy" and "simple", and this feature squarely fits into the "easy" category. But "easened" looks terribly alien to me :-)

File: docs/src/main/asciidoc/logging.adoc [hunk 1]
```diff
@@ -87,8 +87,8 @@ public class MyService {
 <2> Invoke the desired logging methods on the `log` object.
 
 
-[[logging-with-panache]]
-=== Logging with Panache
+[[simplified-logging]]
+=== Simplified Logging
```
Comment (Ladicek) [inline docs/src/main/asciidoc/logging.adoc:91]:
```suggestion
=== Simplified logging
```

We're supposed to use sentence-style capitalizaion for titles now (after I learned to love the Chicago Manual of Style title capitalization :joy:).

**Comment by holly-cummins** [comment]
> As a non-native speaker, I don't have a good alternative to "simplified" :-/ As a huge fan of Rich Hickey's _Simple Made Easy_, I'm well aware of the distinction between "easy" and "simple", and this feature squarely fits into the "easy" category. But "easened" looks terribly alien to me :-)

I agree, "easened" is just weird. If I ruled the world (or this corner of it), I might say "Easified". It's a [mostly-made-up word](https://en.wiktionary.org/wiki/easify), so quite searchable ... and the meaning is clear. However, the I suspect Chicago Manual of Style frowns on made up words. :)

File: docs/src/main/asciidoc/logging.adoc (hunk 1 above)
Comment (holly-cummins) [reply docs/src/main/asciidoc/logging.adoc]: Oh, well spotted. I'd followed the original casing, but in the original Panache was the product name, not a word, which is why it was capitalised.

File: docs/src/main/asciidoc/logging.adoc [hunk 2]
```diff
@@ -59,7 +59,7 @@ In such cases, you need to use a <<logging-apis, logging adapter>> to ensure tha
 In Quarkus, the most common ways to obtain an application logger are by:
 
 * <<declaring-a-loger-field,Declaring a logger field>>
-* <<logging-with-panache,Logging with Panache>>
+* <<simplified-logging,Simplified Logging>>
```
Comment (Ladicek) [inline docs/src/main/asciidoc/logging.adoc:62]:
```suggestion
* <<simplified-logging,Simplified logging>>
```

Sorry, missed this one :-(

**Review by Ladicek — APPROVED** [review]

**Comment by FroMage** [comment]
Well, yeah, from the start, we meant to have lots of extensions `with Panache` which doesn't mean "simplified" at all, but "with style". Most times, "the good style" also means shorter and simpler.

**Pull request merged**